kernel/kernel-impl/src/main/webapp/WEB-INF/content-components.xml

```diff
@@ -282,6 +283,7 @@
         <property name="sakaiFsService" ref="org.sakaiproject.elfinder.SakaiFsService"/>
         <property name="siteService" ref="org.sakaiproject.site.api.SiteService"/>
         <property name="securityService" ref="org.sakaiproject.authz.api.SecurityService"/>
+        <property name="serverConfigurationService" ref="org.sakaiproject.component.api.ServerConfigurationService"/>
```

Comment:
This seems to belong to DropboxToolFsVolumeFactory but I don't see that file as updated. Is there anything missing or can we remove this?

Reply:
https://github.com/sakaiproject/sakai/blob/20e7d2bc2128d7fddbae19374d71f51ecf07aa1c/kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/DropboxToolFsVolumeFactory.java#L32